app/pages/delete-order/contextCreator.test.js

```diff
@@ -0,0 +1,49 @@
+import manifest from './manifest.json';
+import { getDeleteOrderPageContext } from './contextCreator';
+import { baseUrl } from '../../config';
+
+describe('delete-order contextCreator', () => {
+  describe('getContext', () => {

```

Comment:
Very minor, the name of the test suite doesn't match the method name e.g. `getDeleteOrderPageContext`

Reply:
👍